guvnor-structure/guvnor-structure-client/src/main/java/org/guvnor/structure/client/editors/context/GuvnorStructureContextChangeHandler.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.guvnor.structure.client.editors.context;
+
+
+import org.guvnor.structure.repositories.Repository;
+
+public interface GuvnorStructureContextChangeHandler {
```

Comment:
I assume you're using ```Handlers``` to prevent changes in context "leaking" to all Users (as is often the case with CDI Events?)